app/experimenter/nimbus-ui/src/components/PageRequestReview/index.stories.tsx

```diff
@@ -39,6 +39,19 @@ export default {
   decorators: [withLinks],
 };
 
+export const missingRequiredFields = storyWithExperimentProps(
+  {
+    readyForReview: {
+      ready: false,
+      message: {
+        reference_branch: ["This field may not be null."],
```

Comment:
Doesn't affect your PR (thinking outloud here) and I know we're using the same error messages from the server now I wonder if we have a ticket for reviewing any of these error messages, or do you know how we landed on them? I feel like using the word `null` isn't really user friendly. For the risk mitigation questions, I think something like, "This question must be answered." feels a little better...

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/13018240/118038831-39d79380-b335-11eb-8092-bf454c368697.png">

Reply:
Yup, I agree, and to an extent this is initially why I was using client-side messages. I believe these messages are automatically generated from the [serializer](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/api/v5/serializers.py#L533:L533). Not entirely sure what the process is for overriding each one. Could've sworn there was an issue for this but I don't see it so here's a new one: https://jira.mozilla.com/browse/EXP-1315

Reply:
Yeah you can just set this as a parameter to the fields in the serializer, so we can address it on the backend and if the frontend just uses the server messages it'll just pick them up 👍